### PR TITLE
Add additional samples

### PR DESCRIPTION
**Pull request by kaneeldias**
## Purpose
This PR would add the following samples
- Shopify new customers to Outlook email
- Hubspot contacts to Google contacts
- Google drive files to Microsoft one drive files

### CODE REVIEW

File: gdrive-files-to-microsoft-onedrive-files/main.bal [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+configurable string gDriveAccessToken = ?;
+configurable string gDriveFolderId = ?;
+
+configurable string oneDriveAccessToken = ?;
+configurable string oneDrivePath = ?;
+
+configurable boolean filesOverridable = ?;
+
+public function main() returns error? {
+    drive:Client gDriveEndpoint = check new ({auth: {token: gDriveAccessToken}});
```
Comment (anupama-pathirage) [inline gdrive-files-to-microsoft-onedrive-files/main.bal]:
```suggestion
    drive:Client gdrive = check new ({auth: {token: gDriveAccessToken}});
```

Comment (anupama-pathirage) [reply gdrive-files-to-microsoft-onedrive-files/main.bal]: This is the convention we used.

File: gdrive-files-to-microsoft-onedrive-files/main.bal [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    onedrive:Client onedriveEndpoint = check new ({auth: {token: oneDriveAccessToken}});
```
Comment (anupama-pathirage) [inline gdrive-files-to-microsoft-onedrive-files/main.bal]:
```suggestion
    onedrive:Client onedriveEndpoint = check new ({auth: {token: oneDriveAccessToken}});
```
```suggestion
    onedrive:Client oneDrive = check new ({auth: {token: oneDriveAccessToken}});
```

File: gdrive-files-to-microsoft-onedrive-files/main.bal [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+    io:println("Starting to sync files from Google Drive to Microsoft OneDrive...");
```
Comment (anupama-pathirage) [inline gdrive-files-to-microsoft-onedrive-files/main.bal]: this printline is unnecessary?

File: gdrive-files-to-microsoft-onedrive-files/main.bal [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+    string gDriveQuery = string `'${gDriveFolderId}' in parents and trashed = false`;
+    stream<drive:File> filesInGDrive = check gDriveEndpoint->getAllFiles(gDriveQuery);
+
+    io:println("Retrieved files from Google Drive...");
```
Comment (anupama-pathirage) [inline gdrive-files-to-microsoft-onedrive-files/main.bal]: this printline is unnecessary?

File: gdrive-files-to-microsoft-onedrive-files/main.bal [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+    foreach drive:File file in filesInGDrive {
+        string? fileName = file?.name;
+        string? fileId = file?.id;
+        boolean writable = false;
+
+        if fileName !is string || fileId !is string {
```
Comment (anupama-pathirage) [inline gdrive-files-to-microsoft-onedrive-files/main.bal]:
```suggestion
        if fileName is () || fileId is () {
```

File: gdrive-files-to-microsoft-onedrive-files/main.bal [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+            }
+            log:printInfo(string `File ${fileName} uploaded successfully!`);
+        }
+    }
+}
+
+isolated function checkIfFileExistsInOneDrive(string fileName, onedrive:Client onedriveEndpoint) returns boolean|error {
+    stream<onedrive:DriveItemData, onedrive:Error?> searchDriveItems = check onedriveEndpoint->searchDriveItems(
+        fileName);
+    record {|onedrive:DriveItemData value;|}? next = check searchDriveItems.next();
```
Comment (anupama-pathirage) [inline gdrive-files-to-microsoft-onedrive-files/main.bal]: Can we replace these two lines with
`return check searchDriveItems.next() !is ();`

File: gdrive-files-to-microsoft-onedrive-files/main.bal [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+            }
+            writable = !isExistingFile;
+        }
+        if filesOverridable || writable {
+            drive:FileContent|error fileContent = gDriveEndpoint->getFileContent(fileId);
+            if fileContent is error {
+                log:printError("Retrieving file from Google Drive failed!", fileContent);
+                continue;
+            }
+            onedrive:DriveItemData|error uploadFileToFolderByPath = onedriveEndpoint->
```
Comment (anupama-pathirage) [inline gdrive-files-to-microsoft-onedrive-files/main.bal]:
```suggestion
            onedrive:DriveItemData|error driveItemData = onedriveEndpoint->
```

Comment (anupama-pathirage) [reply gdrive-files-to-microsoft-onedrive-files/main.bal]: for simplicity

Comment (kaneeldias) [reply gdrive-files-to-microsoft-onedrive-files/main.bal]: Fixed in [77a8166](https://github.com/ballerina-guides/integration-samples/pull/34/commits/77a8166de8fe1516b5567c05fb5c63274961c532)

File: gdrive-files-to-microsoft-onedrive-files/main.bal (hunk 6 above)
Comment (kaneeldias) [reply gdrive-files-to-microsoft-onedrive-files/main.bal]: [77a8166](https://github.com/ballerina-guides/integration-samples/pull/34/commits/77a8166de8fe1516b5567c05fb5c63274961c532)

File: gdrive-files-to-microsoft-onedrive-files/main.bal (hunk 5 above)
Comment (kaneeldias) [reply gdrive-files-to-microsoft-onedrive-files/main.bal]: [77a8166](https://github.com/ballerina-guides/integration-samples/pull/34/commits/77a8166de8fe1516b5567c05fb5c63274961c532)

File: gdrive-files-to-microsoft-onedrive-files/main.bal (hunk 4 above)
Comment (kaneeldias) [reply gdrive-files-to-microsoft-onedrive-files/main.bal]: Fixed in [77a8166](https://github.com/ballerina-guides/integration-samples/pull/34/commits/77a8166de8fe1516b5567c05fb5c63274961c532)

File: gdrive-files-to-microsoft-onedrive-files/main.bal (hunk 3 above)
Comment (kaneeldias) [reply gdrive-files-to-microsoft-onedrive-files/main.bal]: Fixed in [77a8166](https://github.com/ballerina-guides/integration-samples/pull/34/commits/77a8166de8fe1516b5567c05fb5c63274961c532)

File: gdrive-files-to-microsoft-onedrive-files/main.bal (hunk 2 above)
Comment (kaneeldias) [reply gdrive-files-to-microsoft-onedrive-files/main.bal]: Fixed in [77a8166](https://github.com/ballerina-guides/integration-samples/pull/34/commits/77a8166de8fe1516b5567c05fb5c63274961c532)

File: gdrive-files-to-microsoft-onedrive-files/main.bal (hunk 1 above)
Comment (kaneeldias) [reply gdrive-files-to-microsoft-onedrive-files/main.bal]: Fixed in [77a8166](https://github.com/ballerina-guides/integration-samples/pull/34/commits/77a8166de8fe1516b5567c05fb5c63274961c532)

**Review by anupama-pathirage — APPROVED** [review]

**Pull request merged**